nengo_spa/state.py

```diff
@@ -91,6 +91,13 @@ def add_neuron_input(self):
             warnings.warn("neuron_input already exists. Returning.")
             return self.neuron_input
 
+        if any(isinstance(e.neuron_type, nengo.Direct)
+               for e in self.all_ensembles):
+            raise ValidationError(
+                "Ensembles use Direct neuron type. "
+                "Cannot getneuron output from Direct neurons.",
```

Comment:
And that's why do PR reviews ... copied from the wrong place. Should be fixed now.